models/cow_protocol/cow_protocol_trades.sql

```diff
@@ -0,0 +1,38 @@
+{{ config(
+        alias ='trades'
+        )
+}}
+SELECT *
+FROM
+(
+        SELECT
+            'ethereum' AS blockchain,
+            'CoW Protocol' AS project,
+            '1' AS version,
+            TRY_CAST(date_trunc('DAY', block_time) AS date) AS block_date,
+            block_time,
+            buy_token AS token_bought_symbol,
+            sell_token AS token_sold_symbol,
+            token_pair,
+            units_bought AS token_bought_amount,
+            units_sold AS token_sold_amount,
+            atoms_bought AS token_bought_amount_raw,
+            atoms_sold AS token_sold_amount_raw,
+            usd_value AS amount_usd,
+            buy_token_address AS token_bought_address,
+            sell_token_address AS token_sold_address,
+            trader AS taker,
+            '' AS maker,
+            project_contract_address,
+            tx_hash,
+            trader AS tx_from,
+            receiver AS tx_to,
+            '' AS trace_address,
+            evt_index,
+            'cow_protocol' ||'-'|| tx_hash ||'-'|| IFNULL(evt_index, '') AS unique_trade_id
```

Comment:
@soispoke Is there any guide on how to create this unique_trade_id or can I leave it like this?

Reply:
This looks ok to me, no strict guidelines as long as it makes sense and doesn't create duplicates

Reply:
Can't we just use the OrderUid concatenated with the txhash? Like we do for unique key in the trades table?

Reply:
```suggestion
            'cow_protocol' ||'-'|| tx_hash ||'-'|| order_uid AS unique_trade_id
```